zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -507,6 +507,16 @@ private boolean filterNode(StructuralNode node) {
             try {
                 FilterResult filterResult = scanFilter.isFiltered(node);
                 if (filterResult.isFiltered()) {
+                    try {
+                        HttpMessage msg = node.getHistoryReference().getHttpMessage();
+                        parentScanner.notifyFilteredMessage(msg, filterResult.getReason());
+                    } catch (HttpMalformedHeaderException | DatabaseException e) {
+                        log.error(
```

Comment:
This can be warning (also, for consistency with log at line 554).

Reply:
ok